Makefile

```diff
@@ -83,7 +83,7 @@ REQ_FILES = \
 	requirements/edx/doc \
 	requirements/edx/paver \
 	requirements/edx-sandbox/shared \
-	requirements/edx-sandbox/py35 \
+	requirements/edx-sandbox/base \
```

Comment:
Given that we'll be adding something like `py38.in`/`.txt` soon, maybe we should keep this one as `py35`?